common/utils/fetch_replica_values.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Did we rename this file? Or created a new file with different name?

Reply:
rename only